ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -82,3 +82,6 @@ sunbird_portal_base_url={{sunbird_portal_base_url | d('')}}
 
 #Release-2.6.5
 sunbird_portal_updateLoginTimeEnabled={{sunbird_portal_updateLoginTimeEnabled|lower}}
+
+#Release-2.8.6
+sunbird_portal_slugForProminentFilter={{sunbird_portal_slugForProminentFilter}}
```

Comment:
is this okay to keep this variable empty?
then add 
` sunbird_portal_slugForProminentFilter={{sunbird_portal_slugForProminentFilter | default("")}} `